src/main/scala/cromwell/caching/Cacheable.scala

```diff
@@ -9,8 +12,9 @@ trait Cacheable {
 
   /**
     * Returns hash based on TaskDescriptor attributes.
+    *
     * @return Return hash for related task.
     */
-  def computeHash: Md5sum
+  def computeHash: Future[ExecutionHash]
```

Comment:
Not on this line, but -
I think something's wrong in the type hierarchy here. A Backend isn't "cacheable". A call is cacheable.